docs/partial_source/deep_dive/ivy_frontends_tests.rst

```diff
@@ -490,6 +490,74 @@ This function requires us to create extra functions for generating :code:`shape`
 * We use :code:`helpers.get_dtypes` to generate :code:`dtype`, these are valid numeric data types specifically for Torch.
 * :func:`torch.full` supports :code:`out` so we generate :code:`with_out`.
 
+Testing Without Using Tests Values
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are some places where we cannot test using hypothesis, as some functions have multiple solutions and all those
+solutions can be correct. The values would be different which would cause the tests to fail using hypothesis. What we
+would do in this case is we would set :code:`test_values=False` and test that by reconstructing the output. An example
+is shown below.
+
+.. code-block:: python
+
+    # ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py
+    @handle_frontend_test(
+        fn_tree="torch.linalg.svd",
+        dtype_and_x=helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("float"),
+            min_value=0,
+            max_value=10,
+            shape=helpers.ints(min_value=2, max_value=5).map(lambda x: tuple([x, x])),
+        ),
+        full_matrices=st.booleans(),
+    )
+    def test_torch_svd(
+        *,
+        dtype_and_x,
+        full_matrices,
+        with_out,
+        num_positional_args,
+        as_variable,
+        native_array,
+        frontend,
+        fn_tree,
+        on_device,
+    ):
+        dtype, x = dtype_and_x
+        x = np.asarray(x[0], dtype=dtype[0])
+        # make symmetric positive definite beforehand
+        x = np.matmul(x.T, x) + np.identity(x.shape[0]) * 1e-3
+        ret, frontend_ret = helpers.test_frontend_function(
+            input_dtypes=dtype,
+            as_variable_flags=as_variable,
+            with_out=with_out,
+            num_positional_args=num_positional_args,
+            native_array_flags=native_array,
+            frontend=frontend,
+            fn_tree=fn_tree,
+            on_device=on_device,
+            test_values=False,
+            atol=1e-03,
+            rtol=1e-05,
+            input=x,
+            full_matrices=full_matrices,
+        )
+        ret = [ivy.to_numpy(x) for x in ret]
+        frontend_ret = [np.asarray(x) for x in frontend_ret]
+
+        u, s, vh = ret
+        frontend_u, frontend_s, frontend_vh = frontend_ret
+
+        assert_all_close(
+            ret_np=u @ np.diag(s) @ vh,
```

Comment:
I'd suggest using a different example that is easier to understand, as this may be overwhelming a bit for new reader.